src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./SearchBar.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { far } from "@fortawesome/free-regular-svg-icons";
+import { fab } from "@fortawesome/free-brands-svg-icons";
+
+library.add(fas, far, fab);
+
+const SearchBar = props => {
+  return (
+    <div className="searchBar">
+      <div className="searchBarContainer">
+        <FontAwesomeIcon icon="" />
+        <h1>Instaclone</h1>
+        <div className="searchFormDiv">
+          <form>
+            <input type="text" placeholder="Search" />
```

Comment:
Have the search bar live update the state in `<App />` via `this.setState({ [target.event.name]: target.event.value  })`